Jenkinsfile

```
@@ -80,7 +80,9 @@ pipeline {
                 )
               }
 
-              if (cambpmWithLabels('all-db', 'cockroachdb', 'authorizations')) {
+              // the sidetrack pipeline should be triggered on daily,
+              // or PR builds only, master builds should be excluded
```

@yanavasileva with your suggested changes, we should modify the second line of the comment as well. What do you think?
```suggestion
              // or through PR builds only, master builds should be excluded
```

---

Sorry, I just read the compete sentence, please ignore my comment.